### PR TITLE
update wagmi, rainbowkit and viem

### PR DESCRIPTION
**Pull request by RamRamez**

### CODE REVIEW

File: site/tsconfig.json [hunk 1]
```diff
@@ -23,8 +23,8 @@
       "@components/*": ["components/*"],
       "@commerce": ["../packages/commerce/src"],
       "@commerce/*": ["../packages/commerce/src/*"],
-      "@framework": ["../packages/shopify/src"],
-      "@framework/*": ["../packages/shopify/src/*"]
+      "@framework": ["..\\packages\\shopify\\src"],
+      "@framework/*": ["..\\packages\\shopify\\src/*"]
```
Comment (aminlatifi) [inline site/tsconfig.json:27]: RIs this change necessary for the dependency upgrade? if not please revert it or make it in another PR

**Review by aminlatifi — CHANGES_REQUESTED** [review]

**Review by aminlatifi — APPROVED** [review]

**Review by jainkrati — APPROVED** [review]

**Review by jainkrati — APPROVED** [review]
